src/main/java/core/basesyntax/Data.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Data {
+    private int value;
+    private String type;
```

Comment:
if you want to have "custom object approach", then make it better, make "type" Enum type, instead of String